store/store_test.go

```diff
@@ -1826,34 +1876,19 @@ func (t *remoteRepoTestSuite) TestUbuntuStoreRepositoryDetailsAndChannels(c *C)
 	}
 	result, err := repo.SnapInfo(spec, nil)
 	c.Assert(err, IsNil)
-	c.Assert(n, Equals, 2)
+	c.Assert(n, Equals, 1)
 	c.Check(result.Name(), Equals, "hello-world")
 	c.Check(result.Channels, DeepEquals, map[string]*snap.ChannelSnapInfo{
-		"stable": {
-			Revision:    snap.R(1),
-			Version:     "v1",
+		"latest/stable": {
```

Comment:
This seems pretty slim compared to what we had here earlier. Given the name of this test hinting that this might be the place to test it, do we have a more comprehensive check elsewhere?